src/main/java/core/basesyntax/dao/jdbc/DaoJdbcDriverImpl.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax.dao.jdbc;
+
+import core.basesyntax.dao.DaoDriver;
+import core.basesyntax.exceptions.DataProcessingException;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Driver;
+import core.basesyntax.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@SuppressWarnings("unused")
+@Dao
+public class DaoJdbcDriverImpl implements DaoDriver {
+    @Override
+    public Driver create(Driver item) {
+        String query = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement
+                        = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, item.getName());
+            statement.setString(2, item.getLicenseNumber());
+            statement.execute();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                item.setId(resultSet.getObject("id", Long.class));
+            }
+            return item;
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can't insert object " + item, throwables);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String query = "SELECT * FROM drivers WHERE id = ? AND deleted = false";
```

Comment:
let's not use *. You may add a new columns to the table, so it's better to explicitly write which exactly fields you want to select